common/lib/xmodule/xmodule/tests/test_license.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file needs to be updated later on. I restructured the code to use templates instead of the `html` function, this will need to be refactored in these test cases too. In other words, you can ignore all the `html` tests in this file for now.